pkg/jobrunaggregator/jobruntestcaseanalyzer/cmd.go

```diff
@@ -145,6 +146,8 @@ func (f *JobRunsTestCaseAnalyzerFlags) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.WorkingDir, "working-dir", f.WorkingDir, "The directory to store caches, output, and the like.")
 	fs.DurationVar(&f.Timeout, "timeout", f.Timeout, "Time to wait for analyzing job to complete.")
 	fs.Var(&jobGCSPrefixSlice{&f.JobGCSPrefixes}, "explicit-gcs-prefixes", "a list of gcs prefixes for jobs created for payload. Only used by per PR payload promotion jobs. The format is comma-separated elements, each consisting of job name and gcs prefix separated by =, like openshift-machine-config-operator=3028-ci-4.11-e2e-aws-ovn-upgrade~logs/openshift-machine-config-operator-3028-ci-4.11-e2e-aws-ovn-upgrade")
+
+	fs.StringVar(&f.JobNameFilters, "job-name-filter", f.JobNameFilters, "Applied only when --explicit-gcs-prefixes is not specified.  A comma separated list of substrings used to filter JobNames from the analysis")
```

Comment:
I'd suggest using `StringArrayVar` instead to just get an array, and you can avoid splitting you do later on.

Reply:
Thanks!